django_socio_grpc/tests/grpc_test_utils/fake_grpc.py

```diff
@@ -341,6 +377,11 @@ def fake_method(self, method_name, uri, *args, **kwargs):
         real_method = getattr(handler, method_name)
         self.context = FakeAsyncContext()
 
+        if handler.request_streaming and not handler.response_streaming:
```

Comment:
add comment with link to doc where request_streaming and response_streaming attr are documented please